.gitignore

```diff
@@ -4,4 +4,5 @@
 __pycache__
 
 credentials.json
+settings.json
```

Comment:
Oh. Sorry. I was thinking we could leave this out now. The reason we weren't committing it before was because it contained our credentials. But it may actually be helpful to have us all be on the same settings.json file so I took out the credentials and left this here.

Reply:
Ok. settings.json has a key for terminal path, so I assumed we'd want it left out. But on second thought, our application hasn't needed the key.

Reply:
Ah. Let's put that one in credentials as well. I can ask Ben to move it in his upcoming PR.